modules/rostests/apitests/ntdll/NtApphelpCacheControl.c

```diff
@@ -340,12 +341,21 @@ START_TEST(NtApphelpCacheControl)
     char **argv;
     int argc;
 
+    // Soft get, check and skip, intead of hard load error when executed on NT < 5.2 API (like Windows XP).
+    pNtApphelpCacheControl = (void*)GetProcAddress(GetModuleHandleA("ntdll.dll"), "NtApphelpCacheControl");
+    if (!pNtApphelpCacheControl)
+    {
+        win_skip("NtApphelpCacheControl (NT >= 5.2 API) not available, skipping tests\n");
```

Comment:
Just `skip` ?
Also, the 'skip' macro outputs text that indicates something will be skipped, which you are duplicating now, so the message will look like:
`file(line): : Tests skipped: NtApphelpCacheControl (NT >= 5.2 API) not available, skipping tests`

Reply:
I concur: I simply copied the existing line
`win_skip("RegisterServiceCtrlHandlerExA not available, skipping tests\n");`
I will try `skip()`, as in the rest of this file.
Should I update the other line too, wrt function and/or message?

Reply:
Actually, win_skip is perfectly appropriate here. This should never be missing on ROS, only on Windows.

Reply:
I now had a deeper look:

`NtApphelpCacheControl()`:
Is 5.2, so I agree `win_skip()` applies.

`RegisterServiceCtrlHandlerExA()`:
Is 5.0 (though MS says 5.1), so I agree `skip()` should apply slightly better.
Furthermore, couldn't we just get rid of its `GetProcAddress()`? Otherwise, please suggest a comment about its usefulness.
NB: It doesn't hurt to keep its GPA, but seems overkill.

Reply:
Changing the code below is unnecessary, so don't do it. Keep it short and simple.